administrator/components/com_localise/tmpl/translation/edit.php

```diff
@@ -94,47 +94,39 @@
 Text::script('COM_LOCALISE_CONFIRM_TRANSLATION_SAVE');
 
 Factory::getDocument()->addScriptDeclaration("
-	if (typeof(Localise) === 'undefined') {
-		Localise = {};
-	}
-
-	Localise.language_src = '" . $src . "';
-	Localise.language_dest = '" . $dest . "';
-
 	function returnAll()
 	{
-		$$('i.return').each(function(e){
-			if(e.click)
-				e.click();
-			else
-				e.onclick();
-		});
+		$('.return').trigger('click'); // All with that class will be the clicked 
```

Comment:
Please take off the comment as it is no use.

Reply:
@infograf768 
> Please take off the comment as it is no use.

Done